pkgdown/_pkgdown.yml

```diff
@@ -0,0 +1,42 @@
+url: https://cmap-repos.github.io/cmapplot
+
+home:
+  title: CMAP Themes and Color Palettes
+  description: >
+    Provides themes and color scales for 'ggplot2', based on Chicago
+    Metropolitan Agency for Planning (CMAP) design guidelines.
+
+authors:
+  Chicago Metropolitan Agency for Planning:
+    href: https://www.cmap.illinois.gov
+
+template:
+  params:
+    bootswatch: flatly
+
+reference:
+- title: Theme and Plotting Functions
+- contents:
+  - theme_cmap
+  - finalize_plot
+  - starts_with("geom_")
+  - integer_breaks
+- title: Color Palettes and Gradients
+- contents:
+  - cmap_gradients
+  - cmap_palettes
+  - starts_with("cmap_fill_")
+  - starts_with("cmap_color_")
+  - starts_with("cmap_colour_")
+  - starts_with("viz_")
+- title: Sample Datasets
+- contents:
+  - cluster_jobchange
+  - economy_basic
+  - grp_over_time
+  - percentile_wages
+  - pop_and_laborforce_by_age
+  - recessions
```

Comment:
This organization looks great, except that `recessions` isn't a sample dataset so much as an underlying tool.

Reply:
I'm confused AF. The help says it's "An internal dataset containing a list of all recessions in American history, as recorded by the National Bureau of Economic Research (NBER). As future recessions occur, this file will need to be updated using the script below." And the recessions object looks like a table?

```
> recessions
# A tibble: 33 x 6
   start_char    end_char      start_date end_date   start_num end_num
   <chr>         <chr>         <date>     <date>         <dbl>   <dbl>
 1 June 1857     December 1858 1857-06-01 1858-12-01     1857.   1859.
 2 October 1860  June 1861     1860-10-01 1861-06-01     1861.   1861.
 3 April 1865    December 1867 1865-04-01 1867-12-01     1865.   1868.
 4 June 1869     December 1870 1869-06-01 1870-12-01     1869.   1871.
 5 October 1873  March 1879    1873-10-01 1879-03-01     1874.   1879.
 6 March 1882    May 1885      1882-03-01 1885-05-01     1882.   1885.
 7 March 1887    April 1888    1887-03-01 1888-04-01     1887.   1888.
 8 July 1890     May 1891      1890-07-01 1891-05-01     1890.   1891.
 9 January 1893  June 1894     1893-01-01 1894-06-01     1893    1894.
10 December 1895 June 1897     1895-12-01 1897-06-01     1896.   1897.
# … with 23 more rows
```

Reply:
Are you saying that, even though it's a dataset, its purpose is not for creating sample plots but rather just as an input to the `geom_recessions()` function?

Reply:
Yes, sorry. Your second comment is exactly Right.  Tool = poor word choice.

Reply:
In that case, maybe `recessions` should be an internal-only object, not exported?

Reply:
I will do this.